src/compiler/checker.ts

```diff
@@ -19789,6 +19789,15 @@ namespace ts {
             }
         }
 
+        function checkJSDocParameterTag(node: JSDocParameterTag) {
+            checkSourceElement(node.typeExpression);
+            if (!getParameterSymbolFromJSDoc(node)) {
+                error(node.name,

```

Comment:
This should be a lint-style warning if there is no typeExpression, or if the typeExpression is ignored, the way it is in Typescript.
Until we have a warning infrastructure, I would rather not issue the error in those two cases.